udata/tests/reuse/test_reuse_model.py

```diff
@@ -10,9 +10,15 @@
 from udata.core.user.factories import UserFactory
 
 from .. import TestCase, DBTestMixin
+from udata.frontend.markdown import init_app
 
 
 class ReuseModelTest(TestCase, DBTestMixin):
+    def create_app(self):
```

Comment:
I think we can (we need to) improve that. This change improve run performance but massively reduced test speed (which is already an issue).
Maybe we can disable the behavior on tests and only activate it on demand, when required (in front tests). This is already done for some other behavior with a specific setting like `ENABLE_MARKDOWN_PRERENDERING` which is `True` by default, `False` in testing and ovrriden to `True` when required in tests.